### PR TITLE
Consolidate SASS, clean up, add README

### PR DESCRIPTION
**Pull request by lyzadanger**
This PR consolidates the remaining SASS in the LMS project and adds a basic README explaining styling. With this, we're basically done with this phase of the applied-design project in LMS. (`FilePickerApp` styling still needs to be transitioned, but I see it as part of a larger design-think around assignment configuration).

No visible changes.

Depends on #3962 
Fixes https://github.com/hypothesis/lms/issues/3960

### CODE REVIEW

File: lms/static/styles/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# CSS Styling for LMS
+
+There are currently four CSS stylesheets generated from CSS and SASS source here:
+
+- `frontend_apps.css`, entry point `frontend_apps.scss`: The CSS to style the (preact) front-end components. Components have been converted to use Tailwind utility-first styling in most cases.
```
Comment (robertknight) [inline lms/static/styles/README.md:5]:
```suggestion
- `frontend_apps.css`, entry point `frontend_apps.scss`: The CSS to style the (Preact) front-end components. Components have been converted to use Tailwind utility-first styling in most cases.
```

File: lms/static/styles/lms.scss [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
 @use 'normalize.css/normalize';
 
-@use 'variables' as var;
+$color-text: #202020;
+
+// Typography.
+$sans-font-family: 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande',
+  sans-serif !default;
+
+$normal-font-size: 13px;
```
Comment (robertknight) [inline lms/static/styles/lms.scss:10]: 👀 - If/when we convert the backend-rendered pages to Tailwind, the default font size should obviously be bumped up to the 16px default.

Comment (robertknight) [inline lms/static/styles/lms.scss:10]: 👀 - I wonder if we could just eliminate this and let the font size go up to the 16px default.

File: tailwind.config.mjs [hunk 3]
```diff
@@ -8,6 +8,8 @@ export default {
   content: [
     './lms/static/scripts/frontend_apps/**/*.js',
     './lms/static/scripts/ui-playground/**/*.js',
+    // This script adds a DOM element with the `.browser-check-warning` class
+    './lms/static/scripts/browser_check/index.js',
```
Comment (robertknight) [inline tailwind.config.mjs:12]: Might want to change this to `.../browser_check/**/*.js` so that the CSS doesn't break if the JS gets refactored.

Comment (robertknight) [inline tailwind.config.mjs:12]: Might want to change this to `.../browser_check/**/*.js` so that the CSS doesn't break if the JS gets refactored.

**Review by robertknight — APPROVED** [review]

File: lms/static/styles/lms.scss (hunk 2 above)
Comment (lyzadanger) [reply lms/static/styles/lms.scss:10]: That could have some impact on UI, so I don't want to do it blindly!

**Pull request merged**